### PR TITLE
fix: add title change to merge-event

### PR DESCRIPTION
**Pull request by chyroc**

### CODE REVIEW

File: event_types.go [hunk 1]
```diff
@@ -595,6 +595,10 @@ type MergeEvent struct {
 			Previous int `json:"previous"`
 			Current  int `json:"current"`
 		} `json:"updated_by_id"`
+		Title struct {
+			Previous string `json:"previous"`
+			Current  string `json:"current"`
+		}
```
Comment (svanharmelen) [inline event_types.go]: Please add a `json` tag:
```suggestion
		} `json:"title"`
```

Comment (chyroc) [reply event_types.go]: fixed.

**Comment by chyroc** [comment]
@svanharmelen hi, can this pr be merged

**Comment by chyroc** [comment]
thanks

**Comment by chyroc** [comment]
new release?

**Pull request merged**